ansible_collections/arista/avd/docs/porting-guides/4.x.x.md

```diff
@@ -463,6 +463,24 @@ l3leaf:
     inband_mgmt_description: L2LEAF_INBAND_MGMT
 ```
 
+### Prevent redistributing the MLAG peer link subnets into BGP for all VRFs by default
+
+In AVD 4.0.0, the default value of `redistribute_mlag_ibgp_peering_vrfs` was `true`.
+In AVD 5.0.0, the default value of `redistribute_mlag_ibgp_peering_vrfs` is changed to `false`
```

Comment:
With AVD version 5.0.0 the subnet of VRF L3 peering VLANs between MLAG peers are no longer redistributed into BGP by default. Previously all connected routes were redistributed to BGP.

Reply:
Fixed!